pytorch/training/docker/1.12/py3/cu113/Dockerfile.trcomp.gpu

```diff
@@ -0,0 +1,136 @@
+# docker image region us-west-2
+FROM 763104351884.dkr.ecr.us-west-2.amazonaws.com/pytorch-training:1.12.0-gpu-py38-cu113-ubuntu20.04-sagemaker-v1.0
```

Comment:
Append this statement with `AS sagemaker` since the docker build target is set as `sagemaker`

Reply:
Thanks, updated.